solana/pyth2wormhole/client/src/message.rs

```diff
@@ -0,0 +1,72 @@
+//! Re-usable message scheme for pyth2wormhole
+
+use solana_program::system_instruction;
+use std::{
+    collections::VecDeque,
+    time::{
+        Duration,
+        Instant,
+    },
+};
+
+use crate::ErrBox;
+
+/// One of the accounts tracked by the attestation client.
+#[derive(Clone, Debug)]
+pub struct P2WMessageAccount {
+    /// Unique ID that lets us derive unique accounts for use on-chain
+    pub id: u64,
+    /// Last time we've posted a message to wormhole with this account
+    pub last_used: Instant,
+}
+
+/// An umbrella data structure for tracking all message accounts in use
+#[derive(Clone, Debug)]
+pub struct P2WMessageIndex {
```

Comment:
the name of this struct is confusing because `index` could mean a number, and each account has a numeric id. i suggest `P2WMessageAccountQueue` ?

Reply:
Done